mti_nma/steps/nma/nma.py

```diff
@@ -109,6 +110,48 @@ def run(
         draw_whist(w)
         vmags = get_eigvec_mags(v)
 
+
+        # Working the visualization of eigenvectors on VTK
+        n = polydata.GetNumberOfPoints()
+        writer = vtk.vtkPolyDataWriter()
+
+        for id_mode in range(9):
+
+            # 1st Get eigenvector of interest as a Nx3 array
+            arr_eigenvec = v.T[id_mode,:].reshape(3,-1).T
+
+            # Calculate eigenvector norm
+            arr_eigenvec_norm = np.repeat(np.sqrt(
+                np.power(arr_eigenvec,2).sum(axis=1, keepdims=True)), 3, axis=1)
+
+            # Normalize eigenvectory to unit
+            arr_eigenvec /= arr_eigenvec_norm
+
+            # Convert numpy array to vtk
+            eigenvec = numpy_support.numpy_to_vtk(
+                num_array=arr_eigenvec,
+                deep=True,
+                array_type=vtk.VTK_DOUBLE)
+            eigenvec.SetName('Eigenvector')
+            
+            # Assign eigenvectors as mesh points
+            polydata.GetPointData().AddArray(eigenvec)
+
+            for id_theta, theta in enumerate(np.linspace(0,8*np.pi,64)):
```

Comment:
This sets how many time the animation loops in cycles of `2pi` and the number of timepoints. It might also well been set through parameters..

Reply:
Great - made those parameters too